operator/jsonnet/jsonnetfile.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
this jsonnetfile should only need to point at the base of this repo as a directory type package, like here: https://github.com/coreos/kube-prometheus/blob/master/jsonnetfile.json

the required dependencies are then transitively resolved

Reply:
Here the structure is different, do I tried:

`{
  "version": 1,
  "dependencies": [
    {
      "source": {
        "local": {
          "directory": "../.."
        }
      },
      "version": ""
    }
  ],
  "legacyImports": true
}`

then jb install in `configuration/operator`, then it kind of messed up the directory.
It changed the `jsonnet` dir to a link:
`jsonnet -> /home/frolland/git/go/src/observatorium/configuration`

Reply:
That sounds like a bug in jb

Reply:
make sure to give the package a name still

Reply:
Where should I give a name?

Reply:
`name` is a sibling of `version`

Reply:
Added a name

Reply:
sorry, it's a sibling next to the dependency version, I should have been more clear:

```
{
  "version": 1,
  "dependencies": [
    {
      "source": {
        "local": {
          "directory": "../.."
        }
      },
      "version": "",
       "name": "configuration",
    }
  ],
  "legacyImports": true
}
```

Reply:
I still have the issue with the link. If I have the jsonnetfile.json as above, under operator/jsonnet and then I run jb install, the dir turns to a link to the root, and the jsonnet dir content disappear, along with the jsonnetfile.json

Reply:
Solved with jb built from master